meta/sqlite/sql.go

```diff
@@ -0,0 +1,71 @@
+package sqlite
+
+import (
+	"fmt"
+	"strings"
+)
+
+type QueryOpt func(int) (string, any)
+
+func (q QueryOpt) K(i int) string {
+	k, _ := q(i)
+	return k
+}
+
+func (q QueryOpt) V() any {
+	_, v := q(0)
+	return v
+}
+
+func Option(k, v any) QueryOpt {
+	return func(n int) (string, any) {
+		return fmt.Sprintf(" %s = $%d", k, n+1), v
```

Comment:
Probably me just looking at the examples and seeing they all start at $1.

I did a quick test and was getting this error:

> failed to execute query missing named argument "0"

so it may not be universal within the go driver to use $0.

Reply:
Ah ok thanks for checking (sqlite3 cli allows `.param set $0 foo`). No problem, starting at `$1` is good (I've always used `:key` type notation, so wasn't clear on the numbered notation, assume it has significance when binding values all in one call).